### PR TITLE
[doc] Update serving docs

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

Add Java mode and Binary mode packaging doc

### CODE REVIEW

File: serving/docs/model.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+
+DJL Serving is a high-performance serving system for deep learning models. DJL Serving supports models with:
+
+1. [Python Mode](#python-mode)
+2. [Java Mode](#java-mode)
+3. [Binary Mode](#binary-mode)
+
+In addition to the mode specific files, the `serving.properties` is a configuration file that can be used in all modes. Place `serving.properties` in the same directory with your model file to specify configuration for each model.
+
+In `serving.properties`, you can set number of workers for each model, or you can configure minimum workers and maximum workers differently for GPU and CPU.
```
Comment (frankfliu) [inline serving/docs/model.md:13]: Note: loading model in Python mode is pretty heavy. We recommend to set `mxinWorker` and `maxWorker` to be the same value to avoid unnecessary load and unload.

File: serving/docs/model.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+]
+```
+
+## Binary Mode
+
+This section walks through how to serve Binary model with DJL Serving.
+
+### Provide Model File
+
+For Binary Mode, you just need to place the model file in a folder.
```
Comment (frankfliu) [inline serving/docs/model.md:282]: Binary mode doesn't support pre-processing and post-processing. DJLServing only accept Tensor (NDList/npy/npz) as input and output.

File: serving/docs/model.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+
+DJL Serving supports model artifacts for the following engines:
+
+- MXNet
+- PyTorch (torchscript only)
+- TensorFlow
+- ONNX
+- PadddlePaddle
+
+You can also include a `synset.txt` file in the same directory with your model file to define the labels.
```
Comment (frankfliu) [inline serving/docs/model.md:292]: synset.txt is not supported

File: serving/docs/model.md [hunk 4]
```diff
@@ -0,0 +1,384 @@
...
+
+Next, you need to include a model file. DJL Serving supports model artifacts for the following engines:
+
+- MXNet
+- PyTorch (torchscript only)
+- TensorFlow
+- ONNX
+- PadddlePaddle
+
+You can also include a `synset.txt` file in the same directory with your model file to define the labels.
```
Comment (frankfliu) [inline serving/docs/model.md:183]: You can also include any required artifacts in the model directory. For example, `ImageClassificationTranslator` may need a `synset.txt` file, you can put it in the same ...

File: serving/docs/model.md [hunk 5]
```diff
@@ -0,0 +1,384 @@
...
+    "Persian_cat":0.002232028404250741
+  }
+]
+```
+
+## Java Mode
+
+This section walks through how to serve Java based model with DJL Serving.
+
+### Define a Translator
```
Comment (frankfliu) [inline serving/docs/model.md:161]: We need add a section for using built-in translatorfactory

File: serving/docs/model.md [hunk 6]
```diff
@@ -0,0 +1,384 @@
...
+```
+cd /path/to/model
+./gradlew jar
+```
+
+### Serving Example
+
+Let's run an example where we load a model in Java mode and run inference using the REST API.
+
+#### Step 1: Start Server
```
Comment (frankfliu) [inline serving/docs/model.md:198]: Here you are talking to create translator and package the zip file

File: serving/docs/model.md (hunk 4 above)
Comment (xyang16) [reply serving/docs/model.md:183]: OK

File: serving/docs/model.md (hunk 3 above)
Comment (xyang16) [reply serving/docs/model.md:292]: OK

File: serving/docs/model.md (hunk 2 above)
Comment (xyang16) [reply serving/docs/model.md:282]: OK

File: serving/docs/model.md (hunk 5 above)
Comment (xyang16) [reply serving/docs/model.md]: OK

File: serving/docs/modes.md [hunk 7]
```diff
@@ -0,0 +1,592 @@
...
+
+```
+data = np.zeros((1, 3, 224, 224), dtype=np.float32)
+outfile = TemporaryFile()
+np.savez(outfile, data)
+_ = outfile.seek(0)
+
+response = http.request('POST',
+	'http://localhost:8080/predictions/resnet',
+	headers={'Content-Type':'tensor/ndlist'},
```
Comment (frankfliu) [inline serving/docs/modes.md:582]:
```suggestion
	headers={'Content-Type':'tensor/npz'},
```

**Review by frankfliu — APPROVED** [review]
